annotations/views.py

```diff
@@ -268,12 +268,14 @@ def list_user(request):
     sort_dict = {"user_name":"username", "name":"full_name",
      "aff":"affiliation", "loc":"location"}
 
+    search_term = request.GET.get('search_term', "")
     sort = request.GET.get('sort', 'user_name')
 
     sort_column = sort_dict[sort]
 
-    queryset = VogonUser.objects.exclude(id = -1).order_by(sort_column)
-    paginator = Paginator(queryset, 25)
+    queryset = VogonUser.objects.filter(full_name__icontains = search_term).exclude(id = -1).order_by(sort_column)
```

Comment:
Why not set the default value on line 271 to ``None``, and then only apply the filter if it is set? You can keep it DRY by doing:

```python
queryset = VogonUser.objects.exclude(id=-1).order_by(sort_column)
if search_term:
    queryset = queryset.filter(full_name__icontains=search_term)
```

This way, if there is no search, we see all of the users (except AnonymousUser).